src/domain/event/__tests__/__snapshots__/TicketmasterEventIsEnrolled.test.tsx.snap

```diff
@@ -0,0 +1,14 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`renders snapshot correctly 1`] = `
+<div>
+  <div
+    aria-label="Lataa"
+    class="spinnerWrapper"
+  >
+    <div
+      class="spinner"
+    />
+  </div>
```

Comment:
Spinner is not something we want here :)

Reply:
Hah. That test was actually copied straight from another already existing test, and didn't check it out properly yet. The old test has that spinner snapshot too, so I guess it is just some kind of smoke test or something. Anyway, seems somewhat pointless, so removed that new test.

Reply:
it could also just wait that the spinner is gone!